org.gnome.SimpleScan.json

```diff
@@ -207,15 +207,39 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://downloads.sourceforge.net/sourceforge/hplip/hplip-3.23.12.tar.gz",
-                    "sha256": "a76c2ac8deb31ddb5f0da31398d25ac57440928a0692dcb060a48daa718e69ed",
+                    "url": "https://downloads.sourceforge.net/sourceforge/hplip/hplip-3.24.4.tar.gz",
+                    "sha256": "5d7643831893a5e2addf9d42d581a5dbfe5aaf023626886b8762c5645da0f1fb",
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 1327,
                         "stable-only": true,
                         "url-template": "https://downloads.sourceforge.net/sourceforge/hplip/hplip-$version.tar.gz"
                     }
                 },
+                {
+                    "type": "patch",
+                    "path": "hplip-scan-hpaio-include.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "hplip-hpaio-gcc14.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "hplip-scan-orblite-c99.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "hplip-covscan.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "hplip-pcardext-disable.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "hplip-orblite-use-const.patch"
+                },
```

Comment:
```suggestion
                {
                    "type": "patch",
                    "paths": [
                        "hplip-scan-hpaio-include.patch",
                        "hplip-hpaio-gcc14.patch"
                        "hplip-scan-orblite-c99.patch",
                        "hplip-covscan.patch",
                        "hplip-pcardext-disable.patch",
                        "hplip-orblite-use-const.patch"
                    ]
                },
```

less lines.

Reply:
Since when do we have that?

Reply:
I'd say forever. 

I would have to check the changelog for a more accurate answer.

Reply:
patch is from August 2018. Before 1.0 according to git describe.